### PR TITLE
[NEW][12.0] POS order remove line

### PR DESCRIPTION
**Pull request by robyf70**
Add a new module to remove pos order line

### CODE REVIEW

**Review by eLBati — APPROVED** [review]
Tested

**Comment by legalsylvain** [comment]
Hi @robyf70. Thanks for proposing this module for both 10.0 and 12.0 version.
could you use OCA convention for such PR. 
- recover V10.0 code using that doc :  https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#howto
- make changes 
- commit PR ? 

It allows to : 
- keep history
- make more easy review. (the review is interesting concerning the specific [PORT 12.0] commit)

Thanks.

**Comment by robyf70** [comment]
@legalsylvain Ok! I will do now since the v10 has been merged

**Comment by robyf70** [comment]
@legalsylvain All committed

File: pos_order_remove_line/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+The module adds a the possibility to delete a POS Order Line from the POS interface.
```
Comment (legalsylvain) [inline pos_order_remove_line/readme/DESCRIPTION.rst]: Remove "a" in "a the"

**Review by legalsylvain — APPROVED** [review]
Lgtm. Code review / no test.
Thanks

File: pos_order_remove_line/readme/DESCRIPTION.rst (hunk 1 above)
Comment (robyf70) [reply pos_order_remove_line/readme/DESCRIPTION.rst]: Done :)

**Comment by robyf70** [comment]
@legalsylvain Can you please merge?

**Pull request merged**